src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
+import { getNestedUISchema } from '../helpers';
+
 /*
  * This is the template for each field (which in the schema library means label + widget)
  */
 
 export default function ReviewFieldTemplate(props) {
   const { children, uiSchema, schema } = props;
-  const label = uiSchema['ui:title'] || props.label;
-  const description = uiSchema['ui:description'];
+  const realUISchema = getNestedUISchema(uiSchema);
+  const label = realUISchema['ui:title'] || props.label;
+  const description = realUISchema['ui:description'];
```

Comment:
Destructuring `uiSchema` only to create `realUISchema` is a bit confusing. I don't even know why this is happening, but this approach is preferable:

```diff
diff --git a/src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx b/src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx
index 731db88302..d82a5fcba7 100644
--- a/src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx
+++ b/src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx
@@ -4,7 +4,8 @@ import React from 'react';
  */
 
 export default function ReviewFieldTemplate(props) {
-  const { children, uiSchema, schema } = props;
+  const { children, uiSchema: startingSchema, schema } = props;
+  const uiSchema = getNestedUISchema(startingSchema);
   const label = uiSchema['ui:title'] || props.label;
   const description = uiSchema['ui:description'];
   const textDescription = typeof description === 'string' ? description : null;
```

Reply:
Done!